helpers/Upload.js

```diff
@@ -1,18 +1,22 @@
+import Fs from 'fs';
 import Fetch from 'node-fetch';
 
 const upload = async (posts) => {
-  try {
-    await Fetch(`${process.env.SCRAPER_ADMIN_URL}/upload`, {
-      headers: {
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-      },
-      method: 'POST',
-      body: JSON.stringify(posts),
-    });
-    console.log('Upload successfully!');
-  } catch (err) {
-    console.log(err);
+  const scraper = JSON.parse(Fs.readFileSync('./scraper.json', 'utf8'));
+  if (scraper.id) {
+    try {
+      await Fetch(`${process.env.SCRAPER_ADMIN_URL}/upload`, {
```

Comment:
should create a file that contains all config (from .env, scraper.json...), and another place will import it to use.
this approach will help us to maintain easier.